extensions/amp-story-shopping/0.1/amp-story-shopping-attachment.js

```diff
@@ -53,6 +53,14 @@ export class AmpStoryShoppingAttachment extends AMP.BaseElement {
 
   /** @override */
   buildCallback() {
+    if (this.shoppingTags_.length === 0) {
```

Comment:
Nit: the constructor should never be doing any work (you're losing control over when the work is done), would be cool to move the querySelectors to `buildCallback`